### DISCUSSION TITLE
more sections for Vulnerabilties

### DISCUSSION THREAD

**Discussion by jakuta-tech** [factionsecurity/faction#70] [General]
Hi @summitt  

Is there a way to add more text boxes to the Vulnerabilities section other than the ones that are listed ?

Vulnerability Description: 

Vulnerability Recommendation: 

Details: 

What I'm looking for is to be able to add more sections as needed;  The three you have area already useful but I wanted to add more sections to the vulnerabilities such as below.

Vuln Summary: (same as Description) 

References:

Location: 

Impact: 

PoC: (same as Details)

Remediation: (same as recommendations)

etcetc

Is this possible ?

**Comment by summitt** [comment]
Sorry I missed this, yes you should be able to add these with custom fields. If there are additional rows you want to add to the vuln table then you need to update the 'loop-#' macro. (Example loop-7 if you have 7 rows)
